backend/api/routes/users.py

```
@@ -0,0 +1,785 @@
+    docs = query.stream()
+    feedback_list = [doc.to_dict() for doc in docs]
+
+    return {
+        "feedback": feedback_list,
+        "total": len(feedback_list),
+    }
+
+
+@router.get("/admin/beta/stats")
+async def get_beta_stats(
+    auth_data: Tuple[str, UserType, int] = Depends(require_admin),
```

Ruff (0.14.10)

Unused function argument: `auth_data`

(ARG001)